src/parser/jobs/mnk/modules/Steppies.tsx

```diff
@@ -47,21 +43,20 @@ export default class Steppies extends Module {
 
 	@dependency private checklist!: Checklist
 	@dependency private combatants!: Combatants
-	@dependency private fflogsEvents!: FFLogsEventNormaliser
 	@dependency private suggestions!: Suggestions
 
 	private steppies: Boot[] = []
 
 	protected init(): void {
-		this.addHook('init', () => {
-			this.addHook(this.fflogsEvents.damageEventName, {by: 'player', abilityId: ACTIONS.BOOTSHINE.id}, this.onDamage)
-		})
-		this.addHook('complete', this.onComplete)
+		this.addEventHook('normaliseddamage', {by: 'player', abilityId: ACTIONS.BOOTSHINE.id}, this.onDamage)
+		this.addEventHook('complete', this.onComplete)
 	}
 
-	private onDamage(event: DamageEvent): void {
-		const boot = new Boot(event.criticalHit, this.combatants.selected.hasStatus(STATUSES.LEADEN_FIST.id), event.timestamp)
-		this.steppies.push(boot)
+	private onDamage(event: NormalisedDamageEvent): void {
+		if (event.hits > 0) {
+			const boot = new Boot(event.confirmedEvents[0].criticalHit, this.combatants.selected.hasStatus(STATUSES.LEADEN_FIST.id), event.timestamp)
```

Comment:
thought; drilling down to `confirmedEvents[0]` for a single-target skill is a bit of ick - might be nice to improve in future iterations

Reply:
Yeah, I wasn’t thrilled with this either. I guess the question is whether this approach is more or less bad than trying to come up with a rule for how to mark a multi hit as isCritical/isDH. I’m not really comfortable with a tri-Boolean (eg returning null on multi hits), and it might be a little nonsensical to return isCritical = true if 1 out of 7 AOE hits crits but the other 6 don’t.

Reply:
Yeah, totally gonna agree on that. Something to think about.